packages/react-async/README.md

```diff
@@ -173,6 +173,31 @@ const MyModalComponent = createAsyncComponent({
 });
 ```
 
+#### When to use MyComponent.Preload, MyComponent.Prefetch and MyComponent.KeepFresh
+
+```tsx
+<MyComponent.Preload />
+```
+
+Will only be required when you purposely want to control the pre-loading.
+
+_For example_
+
+```tsx
+const MyComponent = createAsyncComponent({
+  load: () => import(/* webpackChunkName: 'MyComponent' */ './MyComponent'),
+  usePreload: () => usePreload(MyQuery),
+});
+
+<MyComponent.Preload />;
+```
+
+When relying on the library by not including usePreload; in this case, we do not need to have.
```

Comment:
This sentence is a bit unclear and fragemented,  maybe something like:

 When relying on the default behaviour we do not need to render `MyComponent.Preload`

Reply:
Thank you @TheMallen! .. made the change